packages/discord.js/src/sharding/Shard.js

```diff
@@ -406,6 +414,28 @@ class Shard extends EventEmitter {
 
     if (respawn) this.spawn(timeout).catch(err => this.emit('error', err));
   }
+
+  /**
+   * Increments max listeners by one for a given emitter, if they are not zero.
+   * @private
```

Comment:
ig you need to document param here.

Reply:
These methods need to be private just like we have in the BaseClient.
source: https://discord.js.org/#/docs/main/stable/class/BaseClient
I guess we don't need to add these methods to the documentation for these classes either.
I'll leave it to maintainers.